db/migrate/20210309160925_create_katello_content_view_version_import_histories.rb

```diff
@@ -0,0 +1,11 @@
+class CreateKatelloContentViewVersionImportHistories < ActiveRecord::Migration[6.0]
+  def change
+    create_table :katello_content_view_version_import_histories do |t|
+      t.string :path
+      t.text :metadata
+      t.bigint :content_view_version_id
```

Comment:
```suggestion
      t.references :content_view_version, index: { name: :katello_cvv_import_history_cvv_id }
```
Also add a unique index to speed up things and validate duplicate data like
```
    add_index :katello_content_view_version_import_histories, [:content_view_version_id, :path],
              unique: true, name: :katello_cvv_import_history_sat_cvv_id_path
```